### PR TITLE
Responses

### PR DESCRIPTION
**Pull request by Mary0110**

### CODE REVIEW

File: Services/Store/ModsenOnlineStore.Store.Application/Interfaces/CommentInterfaces/ICommentService.cs [hunk 1]
```diff
@@ -5,14 +5,14 @@ namespace ModsenOnlineStore.Store.Application.Interfaces.CommentInterfaces
 {
     public interface ICommentService
     {
-        Task<ResponseInfo<List<GetCommentDto>>> GetAllComments();
+        Task<ResponseInfo> GetAllComments();
```
Comment (elv1sabene) [inline Services/Store/ModsenOnlineStore.Store.Application/Interfaces/CommentInterfaces/ICommentService.cs]: why are there just reponse info, when this method must return data?

**Review by elv1sabene — APPROVED** [review]
please check if all 'Get...' methods returns reponse info with data,

File: Services/Store/ModsenOnlineStore.Store.Application/Interfaces/CommentInterfaces/ICommentService.cs (hunk 1 above)
Comment (Mary0110) [reply Services/Store/ModsenOnlineStore.Store.Application/Interfaces/CommentInterfaces/ICommentService.cs]: We decided to inherit DataResponseInfo from ResponseInfo. That's why all method's declarations return just ResponseInfo

Comment (elv1sabene) [reply Services/Store/ModsenOnlineStore.Store.Application/Interfaces/CommentInterfaces/ICommentService.cs]: Yeah, but how I can get data from controller methods. Lets say I need to check some property in comment. So if you return just ResponseInfo I need to know that I can cast it to DataResponseInfo and eventually do it, which is redundant operation.
Even worse if I do not know that there is such class like DataReponseInfo, then I won't know how to get this comments at all.

Comment (Mary0110) [reply Services/Store/ModsenOnlineStore.Store.Application/Interfaces/CommentInterfaces/ICommentService.cs]: got it, fixed

**Review by elv1sabene — APPROVED** [review]

**Pull request merged**